### PR TITLE
Beautify repo | Closes #3

### LINKED ISSUE #3: Beautify Repo
Add
- [ ] Contributing.md
- [ ] Code of Conduct
- [ ] Repo Cover
- [ ] All Contributors Bot
- [ ] Starware

Take reference from https://github.com/dscnitrourkela/template

### PR DESCRIPTION
**Pull request by CIPHERTron**
1. Add Contributing.md
2. Add Code of Conduct
3. Add Repo Cover
4. Add all contributors bot
5. Add  Starware

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      "name": "DSC NIT Rourkela",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/51153443?v=4",
+      "profile": "http://dscnitrourkela.org",
+      "contributions": [
+        "projectManagement"
+      ]
+    }
+  ],
+  "contributorsPerLine": 7,
+  "projectName": "template",
```
Comment (riteshsp2000) [inline .all-contributorsrc:19]: Change the project name here

**Comment by DesignrKnight** [comment]
Can you add the Issue and PR Template also?
Take reference from the same template repo

**Comment by CIPHERTron** [comment]
> Can you add the Issue and PR Template also?
> Take reference from the same template repo

Yeah Sure🚀

File: .all-contributorsrc (hunk 1 above)
Comment (CIPHERTron) [reply .all-contributorsrc:19]: My bad. Will do it!

**Comment by CIPHERTron** [comment]
All good to go? @DesignrKnight @riteshsp2000

**Review by riteshsp2000 — APPROVED** [review]

**Comment by DesignrKnight** [comment]
@all-contributors add @CIPHERTron for Code

**Pull request merged**